Sticky Sidebar

dhrvsood opened the pull request:
Created temporary filter on left side and sample order summary on right. Will be replaced.

Closes #16.

dhrvsood committed:
Sticky Sidebar

Created temporary filter on left side and sample order summary on right. Will be replaced.

declansullivan commented on app/public/css/menu.css:
```diff
@@ -2,6 +2,10 @@ h2 {
     color: grey;
 }
 
+.container { 
```
Remove empty CSS

declansullivan commented on app/views/menu.ejs:
```diff
@@ -34,15 +31,40 @@
         </div>-->
+
+        <!-- Sample filter (left) *TEMPORARY* -->
+        <div class="filter">
```
These don't need to be <a> tags, since they won't be links in the end of things.  They won't navigate us to a different part of the page, instead they will just filter things out, so these can just be divs.

dhrvsood committed:
Sticky Sidebar edits

Changed a tags in fliter to paragraph tags and moved down.

dhrvsood committed:
Sticky Sides + Order Summary

Added Viren's order summary to sticky right, filter buttons on right are sticky.

NirmalAgnihotri reviewed (approved):
looks good!
